### ISSUE TITLE
Ubuntu 16.04 support

### ISSUE THREAD

**Issue by wizonesolutions** [hashbangcode/vlad#369]
Curious if you fellows have attempted to use Vlad with `bento/ubuntu-16.04` yet at this point.

Wondering where I should look if I want to attempt to get Vlad working with it. If external roles would also have to be updated, etc.

**Comment by philipnorton42** [comment]
I have added Ubuntu 16 support. Needs some more testing/tweaking as I have been concentrating on getting php5.6 support stable in ubuntu 14 (from the back of https://github.com/hashbangcode/vlad/issues/328).

Still, work in progress!

**Comment by wizonesolutions** [comment]
yeah MySQL seems to be sick on the 7.0 build. I will try to fix it because I need 7.0 for a D7 site. Do you know if PHP 7.1 works with Drupal yet?
